components/multiselect/multiselect.tsx

```diff
@@ -297,6 +331,68 @@ export default class MultiSelect<T extends Value> extends React.PureComponent<Pr
             );
         }
 
+        let reactSelectDiv;
+        if (this.props.valueWithImage) {
```

Comment:
Can we build a props object and then use ternary statements for specific properties? I know it's more if's in practice but it will be more readable.

Reply:
@devinbinnie can you advice here?

Reply:
Yeah I'd agree that we do this here, it will definitely be more readable.

Reply:
@devinbinnie Is there a way to do this?
Not to give a prop at all conditionally?

It's not giving the `react-select` prop a different value based on a condition, rather its to either give a few props to it, or not at all.

Reply:
Yes, so instead of having 2 components that give different sets of props, you would specify each prop as such:
```
prop={booleanValue ? thisProp: thatProp}
```